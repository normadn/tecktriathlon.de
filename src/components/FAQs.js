import React from "react";
import ContentWrapper from "./ContentWrapper";

const questionsAndAnswers = [
  {
    question: "Wie teuer ist eine neue Website?",
    answer:
      "Das ist natürlich vom Umfang abhängig. Erfahrungsgemäß zwischen 3-10k €."
  },
  {
    question: "Wie kann ich euch vertrauen, dass ihr meine Kundendaten nicht an Dritte weitergebt?",
    answer: (
      <span>
        Wir stellen höchste Ansprüche an das Thema Datensicherheit. Siehe auch {" "}
        <a
          href="https://amsel.tech/Impressum"
          rel="external noopener"
          target="_blank"
        >
          /Datenschutz
        </a>
      </span>
    )
  },
  {
    question: "Was könnt ihr alles entwickeln?",
    answer: "Websiten, Apps (Mobile, Progressive Web-Apps), Datenbankstrukturen, Analytic-Dashboards und Prozessautomatisierungen."
  },
  {
    question: "Welche Programmiersprachen benutzt ihr dafür?",
    answer:
      "Bei Websites und Web-Apps benutzen wir gerne ReactJS; Im Mobile Bereich können wir sowohl native als auch react-native. Zur Datenanalyse benutzen wir Python und im Backend hauptsächlich Golang."
  },
  {
    question: "Warum heißt ihr Amsel.tech?",
    answer:
      "Wir wohnen im Amselweg 7 in Bissingen an der Teck (fast wie Tech), lieben Vögel und Technik. Alles zusammen ergab den Namen :)"
  } //,
  // {
  //   question: "How do I apply?",
  //   answer:
  //     "Applications will roll out at the end of November, so make sure to stay in the loop by following our social media pages or signing up for our newsletter (link to top). You don't need to be an expert hacker or to have been to a hackathon before - we just want to see what makes you creative and unique!"
  // }
];

const linkCss = {
  textDecoration: "underline",
  color: "#c81c2e",
  fontWeight: "400"
};

const FAQs = () => (
  <section id="faq" css={{ backgroundColor: "#ffffff" }}>
    <ContentWrapper>
      <div
        css={{
          padding: "110px 0 114px 0",
          "@media(max-width: 780px)": {
            padding: "64px 0 120px 0"
          }
        }}
      >
        <h1>Häufig gestellte Fragen</h1>
        <div
          css={{
            paddingTop: "36px",
            display: "grid",
            gridTemplateColumns: "1fr 1fr",
            "@media(max-width: 1190px)": {
              display: "block"
            }
          }}
        >
          {questionsAndAnswers.map(({ question, answer }) => (
            <div
              key={question}
              css={{
                padding: "0px 64px 40px 0px",
                "@media(max-width: 780px)": {
                  paddingRight: "24px"
                }
              }}
            >
              <h2
                css={{
                  paddingBottom: "10px",
                  fontWeight: "500"
                }}
              >
                {question}
              </h2>
              <p
                css={{ fontSize: "14px", color: "#000000", lineHeight: "20px" }}
              >
                {answer}
              </p>
            </div>
          ))}
          <div css={{ paddingTop: "12px" }}>
            <h2 css={{ paddingBottom: "10px", fontWeight: "500" }}>
              Ihr habt noch mehr Fragen?
            </h2>
            <p css={{ fontSize: "14px", color: "#000000", lineHeight: "20px" }}>
              Kontaktiert uns einfach {" "}
              <a
                rel="external noopener"
                target="_blank"
                css={linkCss}
                href="mailto:norman@amsel.tech"
              >
                norman@amsel.tech
              </a>{" "}
              oder unter 07023 9577753
            </p>
          </div>
        </div>
      </div>
    </ContentWrapper>
  </section>
);

export default FAQs;
