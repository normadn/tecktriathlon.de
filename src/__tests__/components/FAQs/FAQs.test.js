import React from "react";
import FAQs from "../../../components/FAQs";
import { shallow } from "enzyme";

describe("FAQs", () => {
  it("renders correctly", () => {
    expect(shallow(<FAQs />)).toMatchSnapshot();
  });
});
