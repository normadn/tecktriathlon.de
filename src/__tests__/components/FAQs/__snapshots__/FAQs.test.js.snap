// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQs renders correctly 1`] = `
<section
  css={
    Object {
      "backgroundColor": "#ffffff",
    }
  }
  id="faq"
>
  <_default>
    <div
      css={
        Object {
          "@media(max-width: 780px)": Object {
            "padding": "64px 0 120px 0",
          },
          "padding": "110px 0 114px 0",
        }
      }
    >
      <h1>
        Frequently Asked Questions
      </h1>
      <div
        css={
          Object {
            "@media(max-width: 1190px)": Object {
              "display": "block",
            },
            "display": "grid",
            "gridTemplateColumns": "1fr 1fr",
            "paddingTop": "36px",
          }
        }
      >
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What is a hackathon?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What is a hackathon?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            A hackathon is a sprint-like event where people from a variety of backgrounds come together to collaborate on software and hardware projects.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="Who is invited?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            Who is invited?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Any college or university student of any skill level! All you need is a passion for tech/data/design and a willingness to learn! We welcome applicants from all fields of study.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="Where is it?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            Where is it?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            QHacks will be taking place at Queen's University in the city of Kingston, Ontario. The event will be held in Mitchell Hall. Don’t worry about getting lost! We will have maps for you and volunteers all over campus to guide the way.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="How much does it cost to attend?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            How much does it cost to attend?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Absolutely nothing! Not one penny. (Remember those?)
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What if I don’t have a team?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What if I don’t have a team?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Not to worry! We'll have a team-formation session prior to the hackathon for you to find people you’d like to work with. There will be plenty of other people in the same situation, and we’ll have you up and running in no time!
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What if I don’t have an idea?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What if I don’t have an idea?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Ideas can come from anywhere! Call a friend, ask your mom, or check out our Facebook group to discuss with other hackers. Sponsors will also provide different project prompts during opening ceremonies to get the juices flowing.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="Can I work on a past project?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            Can I work on a past project?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            No. All projects must be started from scratch at the event.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What if I’m new to programming?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What if I’m new to programming?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            That's okay! A hackathon is a great place to gain valuable experience. There will be tutorials, workshops and mentors to help throughout the event!
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What can I create?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What can I create?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Anything you’d like! Whether it's a new mobile app, website, or a cool project using something from the hardware lab, it’s really up to you. The possibilities are endless.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What should I bring?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What should I bring?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Bring whatever helps you hack! Some important items to bring include a valid form of student ID, laptop and charger, deodorant (arguably mandatory) and maybe a sleeping bag and change of clothes. If you have specific hardware you’d like to work with, feel free to bring it. MLH will also be providing a hardware lab with a wide range of items.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="How will my project be judged?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            How will my project be judged?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Industry experts from a variety of organizations will be coming to QHacks to view your projects. Pitch them your idea, show them a demo, and answer their questions for your chance to win awesome prizes.
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="Is there a Code of Conduct?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            Is there a Code of Conduct?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            <span>
              Yes! Hackers are expected to adhere to the
               
              <a
                href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf"
                rel="external noopener"
                target="_blank"
              >
                Major League Hacking Code of Conduct.
              </a>
            </span>
          </p>
        </div>
        <div
          css={
            Object {
              "@media(max-width: 780px)": Object {
                "paddingRight": "24px",
              },
              "padding": "0px 64px 40px 0px",
            }
          }
          key="What if I'm not from Kingston?"
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            What if I'm not from Kingston?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Encountering a 307 status? We'll be sending busses to Montreal, Ottawa, Toronto and Waterloo. If those aren't convenient for you, we'll be offering travel reimbursements on a case by case basis.
          </p>
        </div>
        <div
          css={
            Object {
              "paddingTop": "12px",
            }
          }
        >
          <h2
            css={
              Object {
                "fontWeight": "500",
                "paddingBottom": "10px",
              }
            }
          >
            Have additional questions?
          </h2>
          <p
            css={
              Object {
                "color": "#000000",
                "fontSize": "14px",
                "lineHeight": "20px",
              }
            }
          >
            Feel free to reach out to us at
             
            <a
              css={
                Object {
                  "color": "#c81c2e",
                  "fontWeight": "400",
                  "textDecoration": "underline",
                }
              }
              href="mailto:hello@qhacks.io"
              rel="external noopener"
              target="_blank"
            >
              hello@qhacks.io
            </a>
             
            or on
             
            <a
              css={
                Object {
                  "color": "#c81c2e",
                  "fontWeight": "400",
                  "textDecoration": "underline",
                }
              }
              href="https://fb.com/qhacks"
              rel="external noopener"
              target="_blank"
            >
              Facebook
            </a>
          </p>
        </div>
      </div>
    </div>
  </_default>
</section>
`;
